cmd/analyze.go

```diff
@@ -862,8 +865,15 @@ func (t *tpccAnalyzer) analyzeTPCC(cloud CloudDetails, machineType string) error
 	if err != nil {
 		return err
 	}
+	runs := goodRuns
```

Comment:
I just realize `runs` seems redundant to `resultFiles` below, right? Maybe we can change the line above to 
```
glob := path.Join(cloud.LogDir(), FormatMachineType(machineType), "tpcc-results.*/success")
```
and only take care of the `includeTpccFail` condition  in the `for _, r := range runs` loop.

Reply:
Nice catch!

Reply:
Updated; sorry for force pushing, mixing git workflows should be discouraged :)

Reply:
The diff after amend is a nice feature: https://github.com/cockroachlabs/cloud-report/compare/7c7c3818febbdd6d07b3014d0319c4c30bd8660b..e5acef3e836e8cb724e72d66bd489dc90baf02b5

Reply:
Re-LGTM!